src/layout/layout.js

```diff
@@ -419,6 +520,11 @@
 
         this.drawer_.addEventListener('keydown', this.keyboardEventHandler_.bind(this));
         this.drawer_.setAttribute('aria-hidden', 'true');
+        // Watch for keyboard events in the navigation.
+        for (var f = 0; f < this.drawerMenuItems_.length; f++) {
+          this.drawerMenuItems_[f].addEventListener('keydown', this.handleMenuNavKeyboardEvent_.bind(this));
+          this.drawerMenuItems_[f].setAttribute('role', 'menuitem');
```

Comment:
Same with the menu role before. Can't developers add this role on their own if they desire it?